docs/generator/buildyaml.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
```

Comment:
Please replace the `\0` with `|` at https://github.com/netdata/netdata/blob/master/docs/generator/buildyaml.sh#L40
It's what was causing issues, when I ran it on my Ubuntu. No idea why I used `\0`.

Reply:
Might it be better to handle that as a separate PR since it appears to break the docs build elsewhere other than just here?

Reply:
I've opened a PR to fix this specifically and tagged you for review.